src/main/scala/newmodel/DocElement.scala

```diff
@@ -0,0 +1,75 @@
+package newmodel
```

Comment:
Take a look at Trees.scala in scalameta. This is a striped version https://raw.githubusercontent.com/metadoc/metadoc/develop/model/README.md

For example Flag would be Modifiers